src/runtime_src/core/pcie/tools/xbutil/xbutil.h

```diff
@@ -969,12 +1012,17 @@ class device {
         pcidev::get_dev(m_idx)->sysfs_get("", "xclbinuuid", errmsg, xclbinid);
         sensor_tree::put( "board.xclbin.uuid", xclbinid );
 
-        // compute unit
-        std::vector<ip_data> computeUnits;
-        if( getComputeUnits( computeUnits ) < 0 ) {
-            std::cout << "WARNING: 'ip_layout' invalid. Has the bitstream been loaded? See 'xbutil program'.\n";
-        }
-        parseComputeUnits( computeUnits );
+        uint32_t kds_mode;
```

Comment:
I see the "classic" version of xbutil being updated, but not version 2 (e.g., Code that exists in src/runtime_src/core/tools/xbutil2).  

Are you planning on updating this code as well?

Reply:
The report CU functionality is WIP. After talked to @saifuddin-xilinx , he would support this in xbutil2.